br/pkg/restore/data.go

```diff
@@ -45,26 +48,36 @@ func RecoverData(ctx context.Context, resolvedTs uint64, allStores []*metapb.Sto
 
 	recovery.makeRecoveryPlan()
 
+	log.Info("recover base alloc id", zap.Uint64("max alloc id", recovery.maxAllocID))
+	if err := recovery.mgr.RecoverBaseAllocID(ctx, recovery.maxAllocID); err != nil {
+		return 0, errors.Trace(err)
+	}
+
 	err = recovery.RecoverRegions(ctx, allStores)
 	if err != nil {
 		return totalRegions, errors.Trace(err)
 	}
 
+	recovery.WaitApply(ctx, allStores)
+	if err != nil {
+		return totalRegions, errors.Trace(err)
+	}
+
 	err = recovery.ResolveData(ctx, allStores, resolvedTs)
 	if err != nil {
 		return totalRegions, errors.Trace(err)
 	}
 	return totalRegions, nil
 }
 
-type RecoveryMeta struct {
-	storeId      uint64
-	recoveryMeta []*recovpb.RegionMeta
+type RegionMeta struct {
```

Comment:
confusing name to have region meta inside region meta

maybe `StoreMeta`? or previous `RecoveryMeta`?

Reply:
agreed.